lib/fluent/plugin/out_forward.rb

```diff
@@ -76,6 +81,19 @@ class ConnectionClosedError < Error; end
     desc 'Compress buffered data.'
     config_param :compress, :enum, list: [:text, :gzip], default: :text
 
+    desc 'The default version of TLS transport.'
+    config_param :tls_version, :enum, list: Fluent::PluginHelper::Socket::TLS_SUPPORTED_VERSIONS, default: Fluent::PluginHelper::Socket::TLS_DEFAULT_VERSION
```

Comment:
Using `tls_` prefix instead of `<tls>` is for avoiding confustion with `<transport tls>`?